.github/workflows/automation-open-pull-request-go-sdk.yaml

```
@@ -0,0 +1,30 @@
+---
+name: Open Pull Request when an `auto-pr` is pushed for go-azure-sdk
+
+on:
+  push:
+    branches:
+      - 'auto-pr/deps/updating-go-azure-sdk-to-**'
+
+jobs:
+  open-pull-request:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
+
+      - name: "open a pull request"
+        id: open-pr
+        run: |
+          $version=$(echo {{ github.ref_name }} | sed "s/auto-pr\/deps\/updating-go-azure-sdk-to-//")
+          $title="dependencies: updating `hashicorp/go-azure-sdk` to `${version}"
```

Assuming this is bash, shouldn't these be?

```suggestion
          version="$(echo {{ github.ref_name }} | sed "s/auto-pr\/deps\/updating-go-azure-sdk-to-//")"
          title="dependencies: updating `hashicorp/go-azure-sdk` to `${version}"
```

(quotes also suggested for safety)